integ/start_with_no_options_test.go

```diff
@@ -28,10 +28,6 @@ func TestStartWorkflowNoOptionsCadence(t *testing.T) {
 }
 
 func doTestStartWorkflowWithoutStartOptions(t *testing.T, backendType service.BackendType) {
-	if !*cadenceIntegTest {
```

Comment:
Noticed that this test was being skipped incorrectly. I removed this and the test passes as expected locally, so I assume it was a mistake in the first place.